packages/components/grid/src/grid.ts

```diff
@@ -304,12 +306,23 @@ export class Grid<T = any> extends ScopedElementsMixin(LitElement) {
         >
           ${this.renderHeader()}
         </thead>
-        <tbody @visibilityChanged=${this.#onVisibilityChanged} part="tbody">
+        <tbody @visibilityChanged=${this.#onVisibilityChanged} id="tbody" part="tbody">
           ${virtualize({
             items: this.view.rows,
             renderItem: (item, index) => this.renderItem(item, index)
           })}
         </tbody>
+        ${this.scrollbar
+          ? html`
+              <tfoot>
```

Comment:
is the `tfoot` the right element for this? (although i don't know what element would fit better semantically 😅 )

Reply:
`<tfoot>` is the only element that is acceptable: the HTML parser is strict when it comes to allowed elements inside of `<table>`. So this is a necessary "hack".